ui/app/AppLayouts/Communities/panels/MembersSettingsPanel.qml

```diff
@@ -197,7 +198,8 @@ SettingsPage {
             if (mode === KickBanPopup.Mode.Kick)
                 root.kickUserClicked(userId)
             else
-                root.banUserClicked(userId)
+                //TODO: reset on close
```

Comment:
Is that TODO still needed and is it gonna be done by another issue?

Reply:
no, forgot to remove
will remove with status-go version update